spotinst/resource_spotinst_elastigroup_aws_test.go

```diff
@@ -4724,3 +4724,64 @@ const testAwaitCapacity_EmptyFields = `
 `
 
 // endregion
+// region OceanAWS: Baseline
```

Comment:
modify the commented line according to the test name

Reply:
updated to region Elastigroup: Logging